.pre-commit-config.yaml

```diff
@@ -17,6 +17,10 @@ repos:
   hooks:
   - id: pyupgrade
     args: [--py38-plus]
+- repo: https://github.com/ikamensh/flynt/
+  rev: ''
```

Comment:
Please specify a revision so that we have deterministic behavior across environments for pre-commit.  0.58 looks like the latest.

Reply:
OK

Reply:
done